cpp/ql/test/query-tests/Likely Bugs/Format/NonConstantFormat/NonConstantFormat.c

```diff
@@ -1,5 +1,9 @@
 extern int printf(const char *fmt, ...);
 
+// For the following `...gettext` functions, we assume that
+// all translations preserve the type and order of `%` specifiers
+// (and hence are safe to use as format strings).  This is
+// assumption is hard-coded into the query.
```

Comment:
I think this comment would be more helpful in the query itself. In any case, please tweak the formatting so the line numbers in the test don't get shifted down. Otherwise it's too hard to review `NonConstantFormat.expected`, and that's the most important file to review.

Reply:
I'm about to push a "new" version of <code>NonConstantFormat.expected</code> with the corrected line numbers.

Reply:
The new version of <code>NonConstantFormat.expected</code> that is in sync with <code>NonConstantFormat.c</code> and <code>test.cpp</code> has been pushed.  Please let me know if you're still encountering problems.